core/src/message.rs

```diff
@@ -92,6 +92,25 @@ pub struct UnregisterUser {
     pub user_id: Id,
 }
 
+/// Register a member of an org on the Radicle Registry with the given ID.
```

Comment:
```suggestion
/// Register a member of an org with the given ID.
```

It is unclear which entity the ID is referring to though, in both versions.

Reply:
This is why I advised to name it handle to make it clear that it is a user.